modules/home/assets/js/components/top-section.js

```diff
@@ -1,7 +1,44 @@
-const TopSection = () => {
+import { Box, Paper } from '@elementor/ui';
+import Typography from '@elementor/ui/Typography';
+import Button from '@elementor/ui/Button';
+import StyledYoutubeIcon from '../icons/youtube-icon';
+
+const TopSection = ( { ctaUrl, videoUrl, embedUrl } ) => {
 	return (
-		<p>Add the TopSection component here</p>
+		<Paper justifyContent="space-between" alignItems="flex-start" elevation={ 0 } sx={ { display: 'flex', flexDirection: 'row', bgcolor: 'theme.palette.background.default', mt: 6 } }>
+			<Box ml={ 4 } my={ 4 }>
+				<Box sx={ { display: 'flex', flexDirection: 'column', width: '85%', mb: 3 } }>
```

Comment:
I often prefer to use a `max-width` in `px`, instead of a `width` as a `%`. 

In this way, the `width` stays full width on mobile, and you don't have to add any responsive styling. 

I think that the `mb` is redundant.

```suggestion
				<Box sx={ { display: 'flex', flexDirection: 'column', maxWidth: '600'  } }>
```